doc/source/User_guide/pyaedt_file_data/project.rst

```diff
@@ -0,0 +1,159 @@
+Project Configuration file
+==========================
+
+The project configuration file is the most comprehensive format that allows to apply a set of variables, materials,
+setup, mesh and boundaries to a new project or save an existing one.
+This file can be created using the following command:
+
+.. code:: python
+
+    from pyaedt import Icepak
+    ipk = Icepak()
+    filename = "test"
+    ipk.export_3d_model(file_name=filename,
+                        file_path=ipk.working_directory,
+                        file_format=".step",
+                        object_list=[],
+                        removed_objects=[])
+    conf_file = ipk.configurations.export_config()
+    ipk.release_desktop()
+
+
+The command is often combined with the ``export_3d_model`` method which allows to export the geometry
```

Comment:
"The command" refers to what command? Should it reference two methods that can be combined?